package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@haventec/common-web-sdk",
-  "version": "1.1.2",
+  "version": "1.2.3",
```

Comment:
Why to 1.2.3? it would be to 1.1.3 or if you want 1.2.0

Reply:
There were features added that involved getting device information, so I think it warrants a minor version upgrade but the patch number should reset to 0

Reply:
The versions have gone a a bit nuts, but we can't go backward